core/controllers/reader_test.py

```diff
@@ -103,6 +106,59 @@ def test_published_explorations_are_visible_to_logged_in_users(self):
             expect_errors=True)
         self.assertEqual(response.status_int, 200)
 
+class ClassifyHandlerTest(test_utils.GenericTestBase):
+    """Test the hander for classification"""
+
+    def setUp(self):
+        super(ClassifyHandlerTest,self).setUp()
+
+    def test_classification_handler(self):
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+
+        # Load demo exploration
+        exp_id = '0'
+        exp_services.delete_demo('0')
+        exp_services.load_demo('0')
+
+        #Reading YAML exploration into a dictionary
+        yaml_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                 '../tests/data/string_classifier_test.yaml')
+        doc_to_label = {}
+        with open(yaml_path, 'r') as yaml_file:
```

Comment:
Perhaps we can move this to setup() (if we are really creating two different tests so that all the necessary setup could be done in one go).